Machine.Specifications.Runner.Resharper.nuspec

```diff
@@ -3,7 +3,7 @@
 	<metadata>
 		<id>Machine.Specifications.Runner.Resharper9</id>
 		<version>$version$</version>
-		<title>Machine.Specifications.Runner.Resharper for ReSharper 2017.3.3</title>
+		<title>Machine.Specifications.Runner.Resharper for ReSharper 2018.1</title>
```

Comment:
You also need to update the Wave version in the nuspec. It should be `12.0` I believe.